cis/encryption.py

```diff
@@ -0,0 +1,52 @@
+import boto3
+import os
+
+from cryptography.hazmat.backends import default_backend
+from cryptography.hazmat.primitives.ciphers import Cipher, algorithms, modes
+
+from cis import settings
+
+
+kms = boto3.client('kms')
+
+
+def encrypt(plaintext):
+    """Encrypt CIS payload using keys derived from KMS."""
```

Comment:
What about adding a couple of more lines in each docstring (enrcypt/decrypt) explaining the args and the general behavior of these functions (although the later is rather self-evident).